setup.cfg

```diff
@@ -0,0 +1,4 @@
+[flake8]
+exclude=invoke/vendor,sites,tests
+ignore=E124,E125,E128,E261,E302,E402,F401,W503,E712,F821
```

Comment:
Basically, you can remove one from the list, then run flake8 to see what lines they apply to.